ngraph_onnx/onnx_importer/ops_bridge/opset_01.py

```diff
@@ -796,6 +796,35 @@ def Split(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]) -> Tup
     return tuple(outputs)
 
 
+def DepthToSpace(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]) -> NgraphNode
+    """Rearranges (permutes) input tensor data from depth into blocks of spatial data.
+
+    Values from the depth dimension (assuming NCHW layout) are moved in spatial blocks to the
+    height and width dimensions.
+
+    :param onnx_node: The ONNX node representing this operation.
+    :param ng_inputs: The input tensors.
+    :return: Tensor with shape [N, C/(blocksize * blocksize), H * blocksize, W * blocksize].
+    """
+    data = ng_inputs[0]
+    block_size = onnx_node.get_attribute_value('blocksize')
+    if block_size is None:
+        raise ValueError('DepthToSpace node (%s): missing required attribute \"blocksize\"',
+                         onnx_node.name)
+    n, c, h, w = 1, 1, 1, 1
```

Comment:
Please add a comment why you're setting these values to 1. "Default values for..."

Reply:
Add more comments.